### ISSUE TITLE
The app crashed when I clicked on "Settings" and "SortBy".

### ISSUE THREAD

**Issue by sagorahmed** [ZoidbergZero/MusicMan#2]
#### Expected behavior
The users should be able to use Settings and SortBy options without getting any trouble.

#### Actual behavior
When I clicked on the "Settings" and "SortBy", then the app crashed.

#### How to reproduce
1. Open the app.
2. Click on ⋮ (Three dots/ Vertical ellipsis).
3. Then try to click "Settings" first and notice the error. After crashing open the app and click on the "SortBy" and notice the error there also. 

* App version: v0.2
* Operating system: Android 6.0.1

#### Logcat of Settings
```

04-24 18:09:56.751  9312  9312 E AndroidRuntime: FATAL EXCEPTION: main

04-24 18:09:56.751  9312  9312 E AndroidRuntime: Process: e.planet.musicman, PID: 9312

04-24 18:09:56.751  9312  9312 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.content.Context.getColor(int)' on a null object reference

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at android.support.v4.content.ContextCompat.getColor(ContextCompat.java:523)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at e.planet.musicman.MainActivity$8.onShow(MainActivity.java:370)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at android.app.Dialog$ListenersHandler.handleMessage(Dialog.java:1694)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7325)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)

04-24 18:09:56.751  9312  9312 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) 
```
#### LogCat of SortBy
``` 
04-24 18:12:11.291  9822  9822 E AndroidRuntime: FATAL EXCEPTION: main

04-24 18:12:11.291  9822  9822 E AndroidRuntime: Process: e.planet.musicman, PID: 9822

04-24 18:12:11.291  9822  9822 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.content.Context.getColor(int)' on a null object reference

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at android.support.v4.content.ContextCompat.getColor(ContextCompat.java:523)

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at e.planet.musicman.MainActivity$5.onShow(MainActivity.java:326)

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at android.app.Dialog$ListenersHandler.handleMessage(Dialog.java:1694)

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7325)

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)

04-24 18:12:11.291  9822  9822 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
```

**Comment by ZoidbergZero** [comment]
Fixed in Rerelease of Version 0.2, Sorry for the Inconvinience.

**Comment by sagorahmed** [comment]
Hello @ZoidbergZero 
Thanks for fixing the issue!!
